cpu/native/include/cpu_conf.h

```diff
@@ -28,7 +28,11 @@ extern "C" {
  * @{
  */
 #ifndef THREAD_STACKSIZE_DEFAULT
+#if (__SIZEOF_POINTER__ == 8)
 #define THREAD_STACKSIZE_DEFAULT            (8192)
+#else
+#define THREAD_STACKSIZE_DEFAULT            (8192)
+#endif
```

Comment:
They are both the same

Reply:
Oops, I didn't save in the IDE before committing... Fixed